app/controllers/api/v1/contacts_controller.rb

```diff
@@ -0,0 +1,19 @@
+module Api
+  module V1
+    class ContactsController < ApiController
+      def create
+        Contact.create!(contact_params)
+        ContactMailer.contact(
+          contact_params[:email],
+          contact_params[:message]
+        ).deliver_later
```

Comment:
this could be done using the `after_create` callback in the model. Not sure what the others think about moving this there because of the skinny-model and skinny-controller pattern. @brunvez wdyt?

Reply:
Never deliver emails or call external apis from a model's callback. You'll regret it once you jump into the rails console session and start creating instances "a mansalva". It's not fun to explain why the customers got random emails